### DISCUSSION TITLE
How to have login page as default ?

### DISCUSSION THREAD

**Discussion by stanthewizzard** [requarks/wiki#5886] [Help / Questions] [answered]
Hello

When I'm accessing wikijs. I land directly to the first page.
Is it possible to land instead on the login page when unloged ?

Thanks

**Comment by ghost** [comment, marked as answer]
yes, it is possible. Its been a long time since I did that, but I think it was by adding a rule into the Guest Group that denies their access to everything. But im not 100% sure. Maybe you can try that out.

**Reply by stanthewizzard** [reply]
thanks I will

**Reply by stanthewizzard** [reply]
it works
